### PR TITLE
created boiler plate html document for css stylings

### PR DESCRIPTION
**Pull request by braddavistech**
This is our boiler plate html doc. Naming conventions were reviewed.

### CODE REVIEW

**Comment by jasehackman** [comment]
crushed it 👍

File: index.html [hunk 1]
```diff
@@ -7,27 +7,58 @@
...
+
+  <article>
+    <header class="article__header">
+      <h1 class="article__header--title">Executive Summary</h1>
+    </header>
+
+    <section class="article__section">
+      
+      <div>
+      <h2 class="section__subtitle"></h2>  <!--Title of article in content block.-->
```
Comment (racheldaniel) [inline index.html]: Can we change "article" to "section" for clarity?

**Comment by braddavistech** [comment]
Fixed the section/article confusion

**Comment by racheldaniel** [comment]
👍 👍

**Comment by jasehackman** [comment]
like a boss

**Comment by danielreedcombs** [comment]
👍

**Pull request merged**